rohit.cpp

```diff
@@ -0,0 +1,57 @@
+#include<bits/stdc++.h>
+#include<stdio.h>
+#include<errno.h>
+#include<sys/types.h>
+#include<sys/socket.h>
+#include<netinet/in.h>
+#include<arpa/inet.h>
+#include<unistd.h>
+using namespace std;
+
+char words[30][30];
+void getwords()
+{
+	for(int i=0;i<6;i++)
+		fgets(words[i],30,stdin);
+}
+
+int main()
+{
+	int csock,len;
+	sockaddr_in client;
+	if((csock=socket(AF_INET,SOCK_STREAM,0))==-1)
+	{
+		perror("Client Socket is not Created\n");
+		exit(-1);
+	}
+	len=sizeof(sockaddr_in);
+	client.sin_family=AF_INET;
+	client.sin_port=htons(1500);
+	client.sin_addr.s_addr=inet_addr("127.0.0.1");
+	bzero(&client.sin_zero,0);
+	
+	if(connect(csock,(sockaddr *)&client,len)==-1)
+	{
+		perror("Connection not Established\n");
+		exit(-1);
+	}
+	
+	getwords();
+	
```

Comment:
remove unnecessary spaces

Reply:
add this into your code `printf("first PR")`

Reply:
ok

Reply:
it's C++ prog. so sorry it would not work.